spring-batch-docs/asciidoc/domain.adoc

```diff
@@ -609,11 +706,50 @@ during the course of execution `StepExecution` and
 `JobExecution` implementations are persisted by
 passing them to the repository:
```

Comment:
This description needs to be updated for this.  The XML snip shows how to define a new `JobRepository`.  The java snip shows how to obtain an instance that has already been configured.  They do two different things.